src/pages/kyunghoon/Comment/CommentList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Comment from './Comment.js';
+
+class CommentList extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      commentList: [],
+      comment: '',
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/commentData.json', {
```

Comment:
`http://localhost:3000`은 생략할 수 있습니다.
```suggestion
    fetch('/data/commentData.json')
```